### DISCUSSION TITLE
How to generate CREATE TABLE IF NOT EXISTS from prisma schema?

### DISCUSSION THREAD

**Discussion by mumincelal** [prisma/prisma#13063] [Q&A] [answered]
When I write a model into schema.prisma file, generated migration sql is becoming `CREATE TABLE <table_name>` but I want generated SQL becomes `CREATE TABLE IF NOT EXISTS <table>`.

I can edit generated SQL file but I don't want to do it. Instead, I want prisma handle it.

Is there any way to do it?

**Comment by austincrim** [comment, marked as answer]
Hey @mumincelal 👋 ,

This is not possible without manually editing the migration file, but it sounds like a cool [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)! Would you mind filling one out?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
